### ISSUE TITLE
Custom Model Architecture Using DarkNet

### ISSUE THREAD

**Issue by RAJA-PARIKSHAT** [AlexeyAB/darknet#8067]
I trained a custom face mask detection model using tiny yolov4 but I didn't make any change in the architecture, let's suppose I want to modify architecture can I do so, and how?
Its like I don't want to use pretrained model rather want to train on custom model from scratch

**Comment by lsd1994** [comment]
Yes you can, you should understand the meaning of each layer first and design your own architecture.
https://github.com/AlexeyAB/darknet/wiki/CFG-Parameters-in-the-different-layers
